change/@microsoft-teams-js-ff815d37-beb3-4aba-a561-f6802208de9d.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Marked `composeExtension` property on `IQueryMessageExtensionResponse` as optional",
```

Comment:
Should this also include the additional input validation and removal of `cardActionsConfig`?

Reply:
`cardActionsConfig` was a separate PR. I'll mention the input validation